assignments/prototypes.js

```diff
@@ -92,6 +105,40 @@
     language: 'Elvish',
   });
 
+  const hero = new Humanoid({
```

Comment:
Great job adding the extra classes, and having them inherit the correct key value pairs!